### ISSUE TITLE
Judge a user by their commits, not just their repos

### ISSUE THREAD

**Issue by TrevorBurnham** [resume/resume.github.com#16]
Plenty of folks have established their bona fides in the open-source software world by committing to existing projects. For instance, Yehuda Katz is probably best known these days for his involvement with RoR and jQuery, both of which he contributed to after they were well-established.

It would be great for Github Resume to list popular projects you don't own on which your commits were accepted; you could be described as a "major" or "minor" contributor, depending on the size of the code you've committed (perhaps relative to other project contributors—say, if you're in the top 10% of contributors according to the amount of current master code you've written, you're considered a major contributor). That's a tricky distinction, obviously; just listing projects you've committed to in order of popularity would be a nice step in the right direction.

**Comment by u2** [comment]
I have added contributions section, but this is imperfect. You can see it http://resume.github.io/?u2 .
There are more works to do:
1. Duplicate repo with own repo (eg: u2/grape-present_cache)
2. Missing org repo (eg: resume/resume.github.com)
3. forked repo

**Comment by markstos** [comment]
Thanks for adding the Contributions section. The "Languages" section is still inaccurate, perhaps reflecting on the repos of languages I host and not those I contribute to. Mine still says "100% Perl", but my contributions of the past two years have been probably about 90% JavaScript, with a little bit of Bash and Perl thrown in. 

See: http://resume.github.io/?markstos

**Comment by u2** [comment]
Yes, as far as I go, I find we are blocked by the Rate Limit Rules of GitHub. I can not get the contributions detail infomation. I will talk to david, I think the only way is add server side for the user OAuth.
